### ISSUE TITLE
Compile aarch64-pc-windows-msvc

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#175620]
Build failure: https://dev.azure.com/monacotools/Monaco/_build/results?buildId=204696&view=logs&j=11d51c58-abd4-52ec-1d76-bd325f6014d1&t=e66b0ccc-53ca-5a06-5d1d-4393c7a5b388&l=421

```
error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX64\\arm64\\link.exe" "/NOLOGO" "C:\\Users\\CLOUDT~1\\AppData\\Local\\Temp\\rustcVHziHD\\symbols.o" "D:\\a\\_work\\1\\s\\cli\\target\\aarch64-pc-windows-msvc\\release\\deps\\code.code.fed869d6-cgu.0.rcgu.o" "/LIBPATH:D:\\a\\_work\\1\\s\\cli\\target\\aarch64-pc-windows-msvc\\release\\deps" "/LIBPATH:D:\\a\\_work\\1\\s\\cli\\target\\release\\deps" "/LIBPATH:C:\\Users\\cloudtest\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\windows_aarch64_msvc-0.36.1\\lib" "/LIBPATH:C:\\Users\\cloudtest\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\windows_aarch64_msvc-0.42.0\\lib" "/LIBPATH:D:\\a\\_work\\1\\s\\cli\\target\\aarch64-pc-windows-msvc\\release\\build\\openssl-sys-cb9af1566673e7ee\\out\\openssl-build\\install\\lib" "/LIBPATH:C:\\Users\\cloudtest\\.rustup\\toolchains\\1.65.0-x86_64-pc-windows-msvc\\lib\\rustlib\\aarch64-pc-windows-msvc\\lib" "C:\\Users\\CLOUDT~1\\AppData\\Local\\Temp\\rustcVHziHD\\libopenssl_sys-d19b252460a88a07.rlib" "C:\\Users\\cloudtest\\.rustup\\toolchains\\1.65.0-x86_64-pc-windows-msvc\\lib\\rustlib\\aarch64-pc-windows-msvc\\lib\\libcompiler_builtins-10214e5a98f5aca5.rlib" "ntdll.lib" "advapi32.lib" "bcrypt.lib" "gdi32.lib" "user32.lib" "crypt32.lib" "ws2_32.lib" "advapi32.lib" "legacy_stdio_definitions.lib" "windows.lib" "ntdll.lib" "windows.lib" "advapi32.lib" "cfgmgr32.lib" "credui.lib" "fwpuclnt.lib" "gdi32.lib" "iphlpapi.lib" "kernel32.lib" "msimg32.lib" "netapi32.lib" "ntdll.lib" "ole32.lib" "oleaut32.lib" "pdh.lib" "powrprof.lib" "psapi.lib" "runtimeobject.lib" "secur32.lib" "shell32.lib" "shlwapi.lib" "synchronization.lib" "user32.lib" "winspool.lib" "ws2_32.lib" "kernel32.lib" "advapi32.lib" "userenv.lib" "kernel32.lib" "ws2_32.lib" "bcrypt.lib" "libcmt.lib" "legacy_stdio_definitions.lib" "/NXCOMPAT" "/LIBPATH:C:\\Users\\cloudtest\\.rustup\\toolchains\\1.65.0-x86_64-pc-windows-msvc\\lib\\rustlib\\aarch64-pc-windows-msvc\\lib" "/OUT:D:\\a\\_work\\1\\s\\cli\\target\\aarch64-pc-windows-msvc\\release\\deps\\code.exe" "/OPT:REF,ICF" "/DEBUG:NONE"
  = note: libopenssl_sys-d19b252460a88a07.rlib(rsa_sup_mul.obj) : error LNK2019: unresolved external symbol _umul128 referenced in function _mul_add_limb
          D:\a\_work\1\s\cli\target\aarch64-pc-windows-msvc\release\deps\code.exe : fatal error LNK1120: 1 unresolved externals
  ```

**Comment by bpasero** [comment]
Hm, I had wrongly assumed this is because of https://github.com/microsoft/vscode/commit/3bee0c9cf80039d540cd6bddca3bc937b3d6df1e but even with the revert, the same error persists: 

https://monacotools.visualstudio.com/DefaultCollection/Monaco/_build/results?buildId=204703&view=logs&j=11d51c58-abd4-52ec-1d76-bd325f6014d1&t=e66b0ccc-53ca-5a06-5d1d-4393c7a5b388

Anyone having an idea?

**Comment by bpasero** [comment]
Oh, this seems to be from the CLI.

**Comment by bpasero** [comment]
I reverted https://github.com/microsoft/vscode/commit/be0f82c93af87f53d38db9397273a8841e60d7e5

**Comment by deepak1556** [comment]
Needs https://github.com/openssl/openssl/commit/eec34033552c810132736c772d37547f097a51b0 which is not available in `1.1.1t`
